test/utils/MockValidationHook.sol

```diff
@@ -5,9 +5,14 @@ import {IHook} from "src/interfaces/IHook.sol";
 import {PackedUserOperation} from "account-abstraction/interfaces/PackedUserOperation.sol";
 
 contract MockValidationHook is IHook {
+    bool internal _verifySignatureReturnValue;
     bytes4 internal _isValidSignatureReturnValue;
     uint256 internal _validateUserOpReturnValue;
```

Comment:
I dislike how each of these returns a different value, I wonder if we should have some middleware or something, because you technically shouldn't be implementing 1271 on the hook